src/github/release.rs

```diff
@@ -12,13 +12,17 @@ impl Tag {
 #[derive(Deserialize, Debug)]
 pub struct Asset {
     pub name: String,
-    #[serde(rename(deserialize = "url"))]
+    #[serde(rename(deserialize = "browser_download_url"))]
```

Comment:
At the start of this project, I was using the `browser_download_url`, but then switched to `url`.
Using the `browser_download_url` would also result in a redirect to the `url`, so here I wouldn't change anything. 

Regarding the `github::download_asset` and the `Accept` header we can just get rid of it since it's not mandatory.

Reply:
I'll test this out, but I'm afraid for the tarball/zipball links dropping `Accept` makes github send back only a json in the response body

Reply:
Ok let me know what you find out

Reply:
With `url` and no `Accept` set, the *ball links work fine, but the asset links refuse to send the file content :/

Reply:
Ok let's revert to your initial implementation that uses `browser_download_url` and `application/vnd.github.raw`

Reply:
Nothing to revert to, I didn't change it 🙂